src/main/java/org/Tema5/Address.java

```diff
@@ -0,0 +1,25 @@
+package org.Tema5;
+
+public class Address {
+
+   final private String street;
+    final private Country country;
```

Comment:
usual convention is modifier first:
```suggestion
   private final String street;
   private final Country country;
```